From ee806b46cff718103355dad1d8eab6c6f3ea9005 Mon Sep 17 00:00:00 2001
From: Sandy lUo <sandy.luo@amlogic.com>
Date: Fri, 27 Jun 2014 19:25:29 +0800
Subject: [PATCH 4403/5965] PD#92955:miracast memory issue

Revert "m102/m101:memory size:save some memory size because HDMI single OUT , camera resolution is 1600x1200"

This reverts commit 96bbdc284f81df74605691566e5a547ee5a65945.
---
 arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd |  7 +++++++
 arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd   | 17 ++++++++++++-----
 2 files changed, 19 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd b/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
index 393997f67dfc..ecd966f294b7 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
@@ -289,6 +289,13 @@ void root_func(){
 		dev_name = "mesonvout";
 		status = "okay";
 	};
+//$$ DEVICE = "mesonvout2"
+//$$ L2 PROP_STR = "status"
+	mesonvout2{
+		compatible = "amlogic,mesonvout2";
+		dev_name = "mesonvout2";
+		status = "okay";
+	};
 
 /// ***************************************************************************************
 ///	-	RTC
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd b/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
index e9d6093cd8c9..f276856386cf 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
@@ -216,8 +216,8 @@ void root_func(){
         compatible = "amlogic,vdin";
         dev_name = "vdin0";
         status = "ok";
-	reserve-memory = <0x01000000>;
-	camera_size = <1600 1200>;//camera is gc2035,the picture resolution is 1600x1200;so 1600x1200x2x4=16M
+	reserve-memory = <0x02700000>;
+	camera_size = <2592 1944>;//2592x1944x2x4=41M
 	 reserve-iomap = "true";
         irq = <115>;
         vdin_id = <0>;
@@ -233,8 +233,8 @@ void root_func(){
         compatible = "amlogic,vdin";
         dev_name = "vdin1";
         status = "ok";
-        reserve-memory = <0x0600000>;
-	miracast_size = <1024 768>;//miracast resolution is 1024x768,1024x768x2x4=6M
+        reserve-memory = <0x01000000>;
+	miracast_size = <1920 1080>;//1920x1080x2x4=17M
 	reserve-iomap = "true";
         irq = <117>;
         vdin_id = <1>;
@@ -250,7 +250,7 @@ void root_func(){
 		compatible = "amlogic,amlvideo2";
 		dev_name = "amlvideo2.0";
 		status = "okay";
-		reserve-memory = <0x0600000>;//miracast resolution is 1024x768,1024x768x1.5x4=6M. NV21,not YUV422 now,so not 1024x768x2x4
+		reserve-memory = <0x01800000>;
 		reserve-iomap = "true";
 	};
 
@@ -289,6 +289,13 @@ void root_func(){
 		dev_name = "mesonvout";
 		status = "okay";
 	};
+//$$ DEVICE = "mesonvout2"
+//$$ L2 PROP_STR = "status"
+	mesonvout2{
+		compatible = "amlogic,mesonvout2";
+		dev_name = "mesonvout2";
+		status = "okay";
+	};
 
 /// ***************************************************************************************
 ///	-	RTC
-- 
2.19.0

